[ADD] add link to "welcome" wiki at start of tuto (#1473)
odoo/documentation: content/developer/howtos/rdtraining/02_setup.rst

Install and configure git
-------------------------

The very first step of the installation process is to install the `git version control system <https://git-scm.com/>`_
because the Odoo source code is managed on GitHub.

> Git can be installed on `Linux <https://git-scm.com/download/linux>`_ , `Windows

Reviewer: ```suggestion
Git can be installed on `Linux <https://git-scm.com/download/linux>`_, `Windows
```

^^^^^^^^^

At first, a simple `print(...)` statement here and there might do the trick. After a while
however, learning how to use a proper debugger will make the experience much better for you.

In the `Debugging page <https://github.com/odoo/enterprise/wiki/Debugging>`__,
we present two of the most common debuggers to track down bugs in Python,
and how you can use them with Odoo.

Mixins
^^^^^^

If you need to interface with common Odoo features such as the chatter, you can rely on
`mixins <https://www.odoo.com/documentation/master/developer/reference/mixins.html>`__.
They are Odoo models exposing useful methods through inheritance.

To learn and play with mixins, visit
`this repository <https://github.com/tivisse/odoodays-2018/>`__.
This module for a plant nursery is training material developed for the OXP 2018.
You don't need to code it on your side. But you can check the presentations in the `/static/pdf`
directory and play with the module to discover some magic features in Odoo.

Security
^^^^^^^^

Every day we hear about data leaks and attacks in the newspapers.
We expect you to be aware and careful about how to avoid security breaches in your future
developments.

> You will define new models during your training, and there are some things
> at which you should pay extra attention:

Reviewer: Since this section will move to advanced topics, the sentence can be changed back to:
```suggestion
You already defined new models in your training. Did you check the access rights and define record rules on them? Are you sure your module is not an open door to SQL injections? Did you use `getattr` or `t-raw` in your code? Here are some things to which you should pay extra attention:
```

- No or wrong access rules on models.
- Introduction of public methods that should actually be private.
- Wrong manipulation of access tokens.
- No or wrong `ir.rules` (restriction rules) on models.
- Introduction of SQL injections.
- etc.

Please check this
`presentation from Olivier Dony (odo)
<https://docs.google.com/presentation/d/1oDINxPtHWz31V8-2W0h2u2ubaKgz9lmbyfx9DJI4lTw/edit>`__
about common mistakes that you should **absolutely** avoid in your future developments.

Also, please always use this
`security checklist
<https://docs.google.com/presentation/d/1oDINxPtHWz31V8-2W0h2u2ubaKgz9lmbyfx9DJI4lTw/edit#slide=id.g2faad955b1_0_6>`__
to double check your development and avoid further embarrassing issues.

> Back to the training!
> 
> Now that your server is running, it's time to start

Reviewer: ```suggestion
Back to the training! Now that your server is running, it's time to start
```
